tool/jt.rb

```diff
@@ -24,6 +24,7 @@
 TRUFFLERUBY_DIR = File.expand_path('../..', File.realpath(__FILE__))
 M2_REPO         = File.expand_path('~/.m2/repository')
 MRI_TEST_CEXT_DIR = "#{TRUFFLERUBY_DIR}/test/mri/tests/cext/c"
+MRI_TEST_CEXT_LIB_DIR = "#{TRUFFLERUBY_DIR}/.ext/c"
```

Comment:
MRI has a `/.ext/-test-/` dir to match their requires. I'm not sure when or why our requires are different.
https://github.com/ruby/ruby/blob/1fe73fc948c5f511d3c6a7b4dfb7fe61348ab169/test/-ext-/hash/test_delete.rb#L3